shared/style/option_menu.css

```diff
@@ -13,3 +13,14 @@ form[role="dialog"][data-subtype="menu"].visible {
   transform: translateY(0);
 }
 
+/* ----------------------------------
+ * Dialogs trick: for further information
```

Comment:
how about:

This CSS disables interaction with the application during dialog animations, to prevent multiple clicks on elements.